src/components/accordion/__tests__/__snapshots__/CdrAccordionGroup.spec.js.snap

```diff
@@ -189,6 +189,41 @@ exports[`CdrAccordionGroup > renders correctly unwrapped 1`] = `
       >
         
         
+      </div>
```

Comment:
I added a new accordion tab to the snapshot to consolidate the code, so the snapshot has changed